common-compose.yml

```diff
@@ -0,0 +1,36 @@
+# MySQL designed for container-native deployment on Joyent's Triton platform.
+
+mysql:
+    image: 0x74696d/mysql:latest
+    mem_limit: 4g
+    restart: always
+    env_file: .env
```

Comment:
Should this really be a dot-file? I ask because it makes it less obvious, but only for people who don't know to look for it.